pkg/ddl/job_worker.go

```diff
@@ -846,7 +846,8 @@ func (w *worker) runOneJobStep(
 		job.State = model.JobStateRunning
 
 		if jobCtx.shouldPollDDLJob() {
-			failpoint.InjectCall("beforePollDDLJob")
+			failpoint.InjectCall("beforePollDDLJob"))
+			jobCtx.logger.Info("beforePollDDLJob", zap.String("job", job.String())
```

Comment:
useless log

you can add field `jobCtx.shouldPollDDLJob()` to the previous log`jobCtx.logger.Info("run one job step", zap.String("job", job.String()))`